### PR TITLE
Versioning and query for Chuan

### PR DESCRIPTION
**Pull request by Galithil**
Chuan needs to be able to access easily some udfs, so I wrote a small script to do just that. 

Also, I am changing the versioning system to be compatible with PEP0440, and try to push this to Pypi.

### CODE REVIEW

File: scripts/limsquery.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+
+
+lims= Lims(BASEURI, USERNAME, PASSWORD)
+
+
+def  get_udfs():
+
+    step_name='End repair, A-tailing and adapter ligation (TruSeq RNA) 4.0'
```
Comment (senthil10) [inline scripts/limsquery.py]: so this script is only for this step ?

Comment (Galithil) [inline scripts/limsquery.py]: You could change that to whichever step you want to get udfs for.

Comment (senthil10) [inline scripts/limsquery.py]: yeah!! I can see this is just a small utility script, but wouldn't it be more cool to take the `step` and `udf` name from command line or a file ? So this script will be untouched. But again, just asking :)

Comment (Galithil) [inline scripts/limsquery.py]: You're right, I could go all fancy and add options, make it work for any
amount of steps or udfs, etc...
But the whole point was to get something out quick so he could get the data
he was looking for. If you wish to implement it properly, be my guest ;)

Comment (senthil10) [inline scripts/limsquery.py]: So this script is only for this step (what Chuan needs) :stuck_out_tongue:

**Comment by senthil10** [comment]
@Galithil : Look fine to me, I can merge it or do you want @pekrau to have look as well ?

**Comment by Galithil** [comment]
http://i.imgur.com/eViMZW6.gif
JUST DO IT

**Comment by senthil10** [comment]
BINGO :+1:

**Pull request merged**